### PR TITLE
russian translation part two

### PR DESCRIPTION
**Pull request by KIVagant**
Null Object
Visitor
Delegation
Template Method
AbstractFactory
Builder
FactoryMethod
Multiton
Pool
Prototype
SimpleFactory
Singleton
StaticFactory
+ Links to ru.wiki in page titles

### CODE REVIEW

File: locale/ru/LC_MESSAGES/Creational/Multiton/README.po [hunk 1]
```diff
@@ -1,64 +1,72 @@
...
+"Last-Translator: Eugene Glotov <kivagant@gmail.com>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language: ru\n"
 
 #: ../../Creational/Multiton/README.rst:2
 msgid "Multiton"
-msgstr ""
+msgstr "`Пул одиночек`_ (`Multiton`__)"
```
Comment (shangguokan) [inline locale/ru/LC_MESSAGES/Creational/Multiton/README.po]: BUILDING ERROR
no english link and no russian link, so:
```
msgstr "Пул одиночек (Multiton)"
```

File: locale/ru/LC_MESSAGES/Structural/README.po [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language: ru\n"
 
 #: ../../Structural/README.rst:2
 msgid "Structural"
 msgstr ""
+"`Структурные шаблоны проектирования <https://ru.wikipedia.org/wiki/"
+"Структурные_шаблоны_проектирования>` (`Structural`__)"
```
Comment (shangguokan) [inline locale/ru/LC_MESSAGES/Structural/README.po]: BUILDING ERROR
 #: ../../Structural/README.rst:2
 msgid "Structural"
 msgstr ""
"`Структурные шаблоны проектирования <https://ru.wikipedia.org/wiki/"
"Структурные_шаблоны_проектирования>`_ (Structural)"

**Comment by shangguokan** [comment]
cool! so fast, after you fix these two points, i'll merge it in

**Comment by shangguokan** [comment]
`` `xxx`__``two underscores is [anonymous-hyperlinks](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#anonymous-hyperlinks)
the first anonymous-hyperlinks in README.rst is english wikipedia. so you use
```
`Команда <https://ru.wikipedia.org/wiki/Команда_(шаблон_проектирования)>`_ (`Command`__)"
```
command will link on the english wikipedia.

`` `xxx <yyy>`_``one underscore is [embedded URIs hyperlinks](http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets)

File: locale/ru/LC_MESSAGES/Structural/README.po (hunk 2 above)
Comment (KIVagant) [inline locale/ru/LC_MESSAGES/Structural/README.po]: What about http://en.wikipedia.org/wiki/Structural_pattern ?

Comment (shangguokan) [inline locale/ru/LC_MESSAGES/Structural/README.po]: ok i'll add these english links in README.rst
    Behavioral pattern
    Creational pattern
    Structural pattern

```
 #: ../../Structural/README.rst:2
 msgid "`Structural`__"
 msgstr ""
"`Структурные шаблоны проектирования <https://ru.wikipedia.org/wiki/Структурные_шаблоны_проектирования>`_ (`Structural`__)"
```

**Comment by KIVagant** [comment]
:D, I like it!

**Comment by shangguokan** [comment]
https://github.com/gelstudios/gitfiti
https://github.com/mappum/gitbanner

**Comment by KIVagant** [comment]
Yeah, I found. Funny

File: locale/ru/LC_MESSAGES/Creational/Multiton/README.po (hunk 1 above)
Comment (KIVagant) [inline locale/ru/LC_MESSAGES/Creational/Multiton/README.po]: Fixed, PR updated

File: locale/ru/LC_MESSAGES/Structural/README.po (hunk 2 above)
Comment (KIVagant) [inline locale/ru/LC_MESSAGES/Structural/README.po]: Fixed, PR updated

**Pull request merged**